.github/workflows/manifests.yml

```diff
@@ -5,7 +5,8 @@ on:
   push:
   pull_request:
     paths:
-      - manifests/**/*.yml
+      - 'manifests/**/*.yml'
+      - '!manifests/templates/**/'
```

Comment:
That will not trigger the workflow on changes in that path, but will still get the list of all .yml files when it does run.

Reply:
@dblock It may actually won't. Since we executed on manifests file that contains `opensearch` prefix like this `ls manifests/**/opensearch*.yml`. All our templates are called just `manifest.yml`

Reply:
Aha! Nice. That works then :)